src/parser.rs

```diff
@@ -3762,9 +3761,18 @@ impl<'a> Parser<'a> {
                     offset = Some(self.parse_offset()?)
                 }
 
-                if offset.is_none() && self.consume_token(&Token::Comma) {
-                    // mysql style LIMIT 10, offset 5
-                    offset = Some(self.parse_offset()?)
+                if dialect_of!(self is MySqlDialect)
```

Comment:
I think the genera pattern in sqlparser is that the `General` dialect supports as much as possible.

Thus can you please also support this syntax with the general as well?

Reply:
Done!